server/lists/adapters/case-summary.js

```diff
@@ -34,6 +34,22 @@ const createAdditionalFields = async (additionalFields = [], fetchList) => {
     return results;
 };
 
+const fetchChoices = async (choices, fetchList) => {
+    if (choices.includes(', ')) {
```

Comment:
If the search string isn't found, Array.prototype.split() will still return an array (with one element), sp you don't need a special-case for if `choices` doesn't include your search string. If you wanted, you could remove this `if` statement.

```js
"foo".split(', ')      // [ "foo" ]
"foo, bar".split(', ') // [ "foo", "bar" ]
```